examples/create_example_nbs.py

```diff
@@ -0,0 +1,59 @@
+import argparse
+from pathlib import Path
+
+import jupytext
+import nbformat
+from nbformat import NotebookNode
+
+instruction: str = "This example requires the following dependencies to be installed:"
```

Comment:
Let's add this directly to the example.py files, then it is also clear that you are expected to install these dependencies when running the script with `python example.py`.